src/DeGiro.ts

```diff
@@ -101,7 +104,7 @@ export class DeGiro implements DeGiroClassInterface {
   login(): Promise<AccountDataType> {
     if (this.jsessionId) return this.loginWithJSESSIONID(this.jsessionId)
     return new Promise((resolve, reject) => {
-      loginRequest({ username: this.username, pwd: this.pwd })
+      loginRequest({ username: this.username, pwd: this.pwd, oneTimePassword: this.oneTimePassword })
```

Comment:
Maybe should put in 3 new lines instead of once

Reply:
Agreed, resolved in upcoming commit.